CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 20.11.5
+* Added new configuration fields.
```

Comment:
"Added new configuration fields to manipulate internal SDK value and key limits"